pkg/v1/service/books/books.go

```diff
@@ -1,10 +1,34 @@
 package books
 
 import (
-    "context"
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/King-Rullmann-Book-Club/books-api/pkg/v1/service/storage"
 )
 
-// GetBook returns the details of a given book ID. 
+// GetBook returns the details of a given book ID.
 func (s *bookSvc) GetBook(ctx context.Context, id string) (Book, error) {
-    return Book{Title: "Perdido Street Station"}, nil
+	rid, err := strconv.ParseUint(id, 10, 0)
+
+	if err != nil {
+		fmt.Errorf("finding %v, %v", id, err)
+		return Book{}, err
+	}
+	return getBookById(uint(rid))
+}
+
+// GetBook returns the details of a given book ID.
+func getBookById(rid uint) (Book, error) {
```

Comment:
Ideally we could replace this with an ORM, but I kinda wanna draw the line there if we're going to keep this 'simple' :D